
package it.kdm.docer.protocollazione;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.10
 * 2021-07-15T14:46:48.652+02:00
 * Generated source version: 2.7.10
 */

@WebFault(name = "WSProtocollazioneProtocollazioneException", targetNamespace = "http://protocollazione.docer.kdm.it")
public class WSProtocollazioneProtocollazioneException_Exception extends Exception {
    
    private it.kdm.docer.protocollazione.WSProtocollazioneProtocollazioneException wsProtocollazioneProtocollazioneException;

    public WSProtocollazioneProtocollazioneException_Exception() {
        super();
    }
    
    public WSProtocollazioneProtocollazioneException_Exception(String message) {
        super(message);
    }
    
    public WSProtocollazioneProtocollazioneException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public WSProtocollazioneProtocollazioneException_Exception(String message, it.kdm.docer.protocollazione.WSProtocollazioneProtocollazioneException wsProtocollazioneProtocollazioneException) {
        super(message);
        this.wsProtocollazioneProtocollazioneException = wsProtocollazioneProtocollazioneException;
    }

    public WSProtocollazioneProtocollazioneException_Exception(String message, it.kdm.docer.protocollazione.WSProtocollazioneProtocollazioneException wsProtocollazioneProtocollazioneException, Throwable cause) {
        super(message, cause);
        this.wsProtocollazioneProtocollazioneException = wsProtocollazioneProtocollazioneException;
    }

    public it.kdm.docer.protocollazione.WSProtocollazioneProtocollazioneException getFaultInfo() {
        return this.wsProtocollazioneProtocollazioneException;
    }
}
